src/OWML.ModHelper.Assets/ModAssets.cs

```diff
@@ -136,10 +133,7 @@ private IEnumerator LoadTexture(TextureAsset modAsset, string imagePath)
 				yield return www;
 				www.LoadImageIntoTexture(texture);
 			}
-			if (texture == null)
-			{
-				_console.WriteLine("Error - Texture is null.", MessageType.Error);
-			}
```

Comment:
Removing error logs...?

Reply:
VS or ReSharper claims texture cannot be null here and I believe it.